contents/java/JavaSynchronization.md

```diff
@@ -15,6 +15,21 @@
 
 Authors: [Boxin](https://github.com/boxin-yang)
 
+<box id="article-toc">
+
+* [Introduction‎](#introduction)
+* [Basic Syntax‎](#basic-syntax)
+	* [Synchronized Statement‎](#synchronized-statement)
+	* [Synchronized Method‎](#synchronized-method)
+* [More Features of Java Synchronization‎](#more-features-of-java-synchronization)
```

Comment:
can u standardise the indentation?

Reply:
changed to 2 spaces instead of using tabs. Thanks